src/components/calloutForm.tsx

```diff
@@ -0,0 +1,225 @@
+import React, { FC, ReactElement } from 'react';
+import { css, SerializedStyles } from '@emotion/core';
+import { Campaign } from '@guardian/apps-rendering-api-models/campaign';
+import { neutral, remSpace, text } from '@guardian/src-foundations';
+import { Button } from '@guardian/src-button';
+import { SvgMinus, SvgPlus } from '@guardian/src-svgs';
+import { Format } from '@guardian/types/Format';
+import { getPillarStyles } from 'pillarStyles';
+import { headline, textSans, body } from '@guardian/src-foundations/typography';
+import { TextInput } from '@guardian/src-text-input';
+import { TextArea } from '@guardian/src-text-area';
+import { RadioGroup, Radio } from "@guardian/src-radio"
+import { FormField } from '@guardian/apps-rendering-api-models/formField';
+import { darkModeCss } from 'styles';
+
+export interface CalloutProps {
+    campaign: Campaign;
+    format: Format;
+}
+
+const calloutStyles = (kicker: string): SerializedStyles => css`
+    border-top: 1px ${neutral[86]} solid;
+    border-bottom: 1px ${neutral[86]} solid;
+    position: relative;
+    margin: ${remSpace[4]} 0 ${remSpace[9]} 0;
+
+    h4 {
+        ${headline.xxxsmall({ fontWeight: 'bold' })};
+        margin: 0;
+    }
+
+    &:not([open]) .is-on, &[open] .is-off {
+        display: none;
+    }
+
+    .is-on, .is-off {
+        position: absolute;
+        transform: translate(0, 50%);
+        bottom: 0;
+    }
+
+    &[open] {
+        background: ${neutral[97]};
+    }
+
+    summary {
+        outline: none;
+        padding: 0;
+        list-style: none;
+        
+        &::-webkit-details-marker {
+            display: none;
+        }
+    
+        .campaign--kicker {
+            display: flex;
+            flex-direction: row;
+    
+            > .campaign--snippet__heading-logo {
+                flex: initial;
+    
+                .speech-bubble {
```

Comment:
With Emotion are we able to avoid these types of deeply nested selectors? Can we apply styles as needed directly to the components? For example, below in `CalloutForm`:

```tsx
const speechBubbleStyles =(kicker: string) => css`
  padding: ${remSpace[3]};
  position: relative;
  color: ${neutral[100]};
  background-color: ${kicker};
  margin-bottom: ${remSpace[9]};
  min-width: 5.5rem;

  &::after {
    content: '';
    width: 1.25rem;
    height: 1.375rem;
    border-radius: 0 0 1.125rem;
    position: absolute;
    bottom: -0.75rem;
    left: 0.625rem;
    color: ${neutral[100]};
    background-color: ${kicker};
  }
`;

const headingStyles = css`
  ${headline.xxxsmall({ fontWeight: 'bold' })};
  margin: 0;
`;

const Logo = () =>
  <div css={css`flex: initial;`}>
    <div css={speechBubbleStyles(kicker)}>
      <h4 css={headingStyles}>Take part</h4>
    </div>
  </div>
```

I think this approach will allow us to split the UI into smaller components. Also, applying styles directly like this without relying on tag and class selectors will make it easier to scope the styles to specific elements, in addition to altering the structure if needed.